lsp-pwsh.el

```diff
@@ -256,25 +260,24 @@ Must not nil.")
 (defun lsp-pwsh--extra-init-params ()
   "Return form describing parameters for language server.")
 
-(defun lsp-pwsh--apply-code-action-edits (action)
+(lsp-defun lsp-pwsh--apply-code-action-edits ((&pwsh:ApplyCodeActionEdits :command :arguments))
```

Comment:
This should be `&Command`
https://github.com/microsoft/language-server-protocol/blob/gh-pages/_specifications/specification-3-16.md#-command-

Reply:
Thanks for noticing that